notebooks/openvino-tokenizers/openvino-tokenizers.ipynb

```diff
@@ -18,6 +18,7 @@
     "    - [Convert Tokenizer from HuggingFace Hub with CLI Tool](#Convert-Tokenizer-from_HuggingFace-Hub-with-CLI-Tool)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    ov.save_model(combined_model, model_dir / "combined_openvino_model.xml")
</pre><p>I think in this sample also good to demonstrate how model inputs changed in combined model  (1 input that expect text instead of original model inputs)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Added